FiftyOne.Pipeline.Engines.FiftyOne/FlowElements/ShareUsageBase.cs

```
@@ -775,6 +783,12 @@ private static bool IsLocalHost(IPAddress address)
         /// </param>
         private void ProcessData(IFlowData data)
         {
+            // Check if we're still in recovery period
+            if (!_recoveryStrategy.MayTryNow(out CachedException cachedException))
+            {
+                return;
```

Since this element is going to silently ignore the unavailability,
the design of 
https://github.com/51Degrees/pipeline-dotnet/blob/e54eab5fb2353e0cc27b0e66f897001c87a4bfc0/FiftyOne.Pipeline.Core/FailHandling/Facade/WindowedFailHandler.cs#L155

may no longer be appropriate (as we shouldn't rely on exceptions for non-critical paths).
_( * If we were to use it in this element -- see the comment above )_

At the least, we should consider making the thrown exception not directly tied to Pipeline as a whole.